engine/api/purge/purge_v2.go

```diff
@@ -363,29 +367,40 @@ func DeleteArtifactsFromRepositoryManagerV2(ctx context.Context, db gorp.SqlExec
 	props.AddProperty("ovh.to_delete", "true")
 	props.AddProperty("ovh.to_delete_timestamp", strconv.FormatInt(time.Now().Unix(), 10))
 
+	results := make(chan bool, len(runResults))
```

Comment:
Maybe using a wait group here is easier to maintain.

Reply:
fixed